includes/admin/builder/class-evf-builder-settings.php

```diff
@@ -254,41 +254,97 @@ public function output_content() {
 			if ( preg_match( '/connection_/' , $connection_id ) ) {
 				echo '<div class="evf-content-email-settings-inner" data-connection_id='.$connection_id.'>';
 
-				everest_forms_panel_field(
-					'text',
-					'settings[email]['.$connection_id.']',
-					'connection_name',
-					$this->form_data,
-					'',
-					array(
-						'default' => isset( $settings['email'][$connection_id]['connection_name'] ) ? $settings['email'][$connection_id]['connection_name'] : __('Admin Notification', 'everest-forms'),
-						'class'   => 'everest-forms-email-name',
-					)
-				);
-				everest_forms_panel_field(
-					'text',
-					'settings[email]['.$connection_id.']',
-					'evf_to_email',
-					$this->form_data,
-					__( 'To Address', 'everest-forms' ),
-					array(
-						'default' => isset( $settings['email'][$connection_id]['evf_to_email'] ) ? $settings['email'][$connection_id]['evf_to_email'] : get_option( 'admin_email' ),
-						'smarttags'  => array(
-							'type'   => 'fields',
-							'form_fields' => 'email',
-						),
-					)
-				);
-				everest_forms_panel_field(
-					'text',
-					'settings[email]['.$connection_id.']',
-					'evf_from_name',
-					$this->form_data,
-					__( 'From Name', 'everest-forms' ),
-					array(
-						'default' => isset( $settings['email'][$connection_id]['evf_from_name'] ) ? $settings['email'][$connection_id]['evf_from_name'] : get_bloginfo( 'name', 'display' ),
-						'smarttags'  => array(
+			everest_forms_panel_field(
```

Comment:
Why only indentation changes are reflected as diff. Please stick with WP coding standard not space for indentation do use tab indentation?

Reply:
Fixed :)